lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -952,6 +953,12 @@ def _create_deployment_parameters(
             parameters=parameters,
         )
 
+        # dump arm_template and arm_parameters to file
+        template_dump_path = environment.log_path / f"{environment.name}_template.txt"
```

Comment:
The names can be `arm_template.json` and `arm_template_parameters.json`. These files are in the env folder already, so the env name is not necessary.